nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -128,14 +128,15 @@ def extcap_interfaces():
         Wireshark-related method that returns configuration options.
         :return: string with wireshark-compatible information
         """
-        # TODO: Detect connected sniffers and print one interface per each sniffer
         res = []
         res.append("extcap {version=1.0}{help=https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver}{display=nRF 802.15.4 sniffer}")
-        res.append("interface {value=nrf802154}{display=nRF 802.15.4 sniffer}")
-        res.append("control {number=%d}{type=selector}{display=Channel}{tooltip=IEEE 802.15.4 channel}" % Nrf802154Sniffer.CTRL_ARG_CHANNEL)
+        for port in comports():
+            if port.vid == Nrf802154Sniffer.NORDICSEMI_VID and port.pid == Nrf802154Sniffer.SNIFFER_802154_PID:
+                res.append("interface {value=%s}{display=nRF 802.15.4 sniffer}" % (port.device,))
+                res.append("control {number=%d}{type=selector}{display=Channel}{tooltip=IEEE 802.15.4 channel}" % Nrf802154Sniffer.CTRL_ARG_CHANNEL)
```

Comment:
Add the ```control``` and ```value``` elements once, not one for each interface (comport).

Running ```./nrf802154_sniffer.py --extcap-interfaces``` should print something like this:
```
interface {value=/dev/cu.usbmodemF4A2F9F7EFA42}{display=nRF 802.15.4 sniffer}
interface {value=/dev/cu.usbmodemE9A9CB8D83252}{display=nRF 802.15.4 sniffer}
control {number=0}{type=selector}{display=Channel}{tooltip=IEEE 802.15.4 channel}
value {control=0}{value=11}{display=11}
value {control=0}{value=12}{display=12}
[...]
```

Reply:
Thanks, fixed